src/Couchbase.Lite.Tests.Shared/DocumentTest.cs

```diff
@@ -2100,6 +2100,70 @@ public void TestRevisionIDExistingDoc()
             }
         }
 
+        [Fact]
+        public void TestTypesInDocumentToJSON()
+        {
+            var dic = PopulateDictData();
+            using (var md = new MutableDocument("doc1")) {
+                foreach (var item in dic) {
```

Comment:
What is going on here?  Can't this just call `SetValue`?

Reply:
I can also do SetValue with a dictionary, but since I am combining test types at once instead of each type individually, I want to keep using the specific set method for different type to distinguish from setValue with the whole dictionary....

Reply:
You can loop still, and just call `SetValue` right?  I don't see the need for these dozens of lines of code.

Reply:
Done.